models/varnet/varnet.py

```diff
@@ -301,7 +315,7 @@ def add_model_specific_args(parser):
         return parser
 
 
-def create_trainer(args):
+def create_trainer(args, logger=True):
```

Comment:
Can you just remove the logger argument completely? It's not being used in the function anyway

Reply:
Sure, no problem. Yes, I forgot to make the U-Net model compatible as well but will try to edit accordingly. I'll remove the `logger` argument as well.

Reply:
Added a new commit with the changes